Sources/SourceKitBazelBSP/RequestHandlers/SKOptions/BazelTargetAquerier.swift

```diff
@@ -68,11 +67,15 @@ final class BazelTargetAquerier {
         }
 
         // Run the aquery on the special index output base since that's where we will build at.
-        let output = try commandRunner.bazelIndexAction(initializedConfig: config, cmd: cmd)
+        let output = try commandRunner.bazelIndexAction(initializedConfig: config, cmd: cmd).asData
 
-        queryCache[cmd] = output
+        let parsedOutput = try BazelProtobufBindings.parseActionGraph(data: output)
 
-        return output
+        logger.debug("actionGraphContainer contains \(parsedOutput.actions.count, privacy: .public) actions.")
```

Comment:
```suggestion
        logger.debug("ActionGraphContainer count: \(parsedOutput.actions.count)")
```

We should keep the logs private as that's the right way to use OSLog (although I agree it's a bit annoying)